bedevere/news.py

```diff
@@ -52,6 +60,8 @@ async def check_news(gh, pull_request, files=None):
         if util.skip("news", issue):
             status = SKIP_LABEL_STATUS
         else:
+            await gh.post(pull_request['issue_comment_url'],
```

Comment:
Add before this:

```
if pull_request['author_association'] == 'NONE':
```

Reply:
`pull_request['issue_comment_url']` is not the right POST url.

API doc: https://docs.github.com/en/rest/reference/issues#create-an-issue-comment

The path is `/repos/{owner}/{repo}/issues/{issue_number}/comments`, so this should look like:
```suggestion
            await gh.post(f"{pull_request['issue_url']}/comments",
```